WatchGuard patches to sources for traceroute-6.1.3 as of
Mon May  9 14:02:33 PDT 2022
The patches shown here have been applied to source .tar.gz 
files supplied with the WatchGuard Open Source Archive.

==========================================================================
--- traceroute-6.1.3/traceroute.c.orig	2022-05-09 14:02:33.258474327 -0700
+++ traceroute-6.1.3/traceroute.c	2022-05-09 14:02:33.334471245 -0700
@@ -2492,7 +2492,7 @@
     int len = ntohs(getshort(ans+off));
     int retval = off+len+2;
     int i,j;
-    char name[256];
+    static char name[256];
 
     off += 2;
     switch (typ) {
